From e7c116e1c34fe8b7a6d0409d27f12487d5bfd640 Mon Sep 17 00:00:00 2001
From: Richard Hu <richard.hu@technexion.com>
Date: Thu, 5 Nov 2015 15:12:28 +0800
Subject: [PATCH 2/3] edm-cf-imx6: put zImage and dtb into root directory
 instead of /boot in 1st FAT partition to follow the
 rule in image_types_fsl.bbcleass in yocto

---
 board/technexion/edm-cf-imx6/edm-cf-imx6.c   |    8 ++++----
 board/technexion/picosom-imx6/picosom-imx6.c |    4 ++--
 include/configs/edm-cf-imx6.h                |    2 +-
 include/configs/picosom-imx6.h               |    2 +-
 4 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/board/technexion/edm-cf-imx6/edm-cf-imx6.c b/board/technexion/edm-cf-imx6/edm-cf-imx6.c
index a0d8f03..7529be6 100644
--- a/board/technexion/edm-cf-imx6/edm-cf-imx6.c
+++ b/board/technexion/edm-cf-imx6/edm-cf-imx6.c
@@ -666,16 +666,16 @@ int misc_init_r(void)
 			case EDM1_CF_IMX6_SOM:
 			case EDM2_CF_IMX6_SOM:
 				if (is_cpu_type(MXC_CPU_MX6Q) || is_cpu_type(MXC_CPU_MX6D))
-					setenv("fdt_file", "boot/imx6q-edm1-cf.dtb");
+					setenv("fdt_file", "imx6q-edm1-cf.dtb");
 				else
-				setenv("fdt_file", "boot/imx6dl-edm1-cf.dtb");
+					setenv("fdt_file", "imx6dl-edm1-cf.dtb");
 				break;
 			case WANDBOARD_B1_SOM:
 			case WANDBOARD_C1_SOM:
 				if (is_cpu_type(MXC_CPU_MX6Q) || is_cpu_type(MXC_CPU_MX6D))
-					setenv("fdt_file", "boot/imx6q-wandboard.dtb");
+					setenv("fdt_file", "imx6q-wandboard.dtb");
 				else
-					setenv("fdt_file", "boot/imx6dl-wandboard.dtb");
+					setenv("fdt_file", "imx6dl-wandboard.dtb");
 				break;
 			}
 		}
diff --git a/board/technexion/picosom-imx6/picosom-imx6.c b/board/technexion/picosom-imx6/picosom-imx6.c
index 9298f94..00648f4 100644
--- a/board/technexion/picosom-imx6/picosom-imx6.c
+++ b/board/technexion/picosom-imx6/picosom-imx6.c
@@ -669,9 +669,9 @@ int misc_init_r(void)
 	if ((s = getenv ("fdt_file_autodetect")) != NULL) {
 		if (strncmp (s, "off", 3) != 0) {
 			if (is_cpu_type(MXC_CPU_MX6Q) || is_cpu_type(MXC_CPU_MX6D))
-				setenv("fdt_file", "boot/imx6q-picosom.dtb");
+				setenv("fdt_file", "imx6q-picosom.dtb");
 			else
-				setenv("fdt_file", "boot/imx6dl-picosom.dtb");
+				setenv("fdt_file", "imx6dl-picosom.dtb");
 		}
 	}
 
diff --git a/include/configs/edm-cf-imx6.h b/include/configs/edm-cf-imx6.h
index bc3a2be..666ed40 100644
--- a/include/configs/edm-cf-imx6.h
+++ b/include/configs/edm-cf-imx6.h
@@ -140,7 +140,7 @@
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
 "script=boot.scr\0" \
-"image=boot/zImage\0" \
+"image=zImage\0" \
 "console=" __stringify(CONFIG_DEBUG_TTY) "\0" \
 "splashpos=m,m\0" \
 "fdt_high=0xffffffff\0" \
diff --git a/include/configs/picosom-imx6.h b/include/configs/picosom-imx6.h
index e93f0e1..f2734df 100644
--- a/include/configs/picosom-imx6.h
+++ b/include/configs/picosom-imx6.h
@@ -137,7 +137,7 @@
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	"script=boot.scr\0" \
-	"image=boot/zImage\0" \
+	"image=zImage\0" \
 	"console=ttymxc0\0" \
 	"splashpos=m,m\0" \
 	"fdt_high=0xffffffff\0" \
-- 
1.7.9.5

